Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.21
+
+RUN apt-get update \
+  && apt-get install --no-install-recommends --no-install-suggests -y ca-certificates jq yq \
+  && rm -rf /var/lib/apt/lists/*
+
+COPY <<EOF /entrypoint.sh
```

Comment:
This is not a blocker by any means but rather for my understanding. 
Heredocs in dockerfiles is still a bit of a novel way of adding a script to a dockerfile. We do lose out on some coding features by not having it in a standalone .sh file. What was the purpose of adding it here?

Reply:
Still WiP while I figure out what's needed exactly. It was previously in the nginx repo alongside the tests and I wanted to minimize the number of files needed.